### DISCUSSION TITLE
trying to invite new follower to dexcom share

### DISCUSSION THREAD

**Discussion by JGroucutt** [NightscoutFoundation/xDrip#2557] [Q&A]
I've been trying to add a dexcom follower using the features in the app, but each time i get a message saying "Failed to invite follower". Does anyone know what is failing, and how i might fix it? (been using what i assume is the correct information)

**Comment by Navid200** [comment]
So, you are using xDrip as a collector and you want someone to follow and you want to use Dex share?

Would you please explain why you cannot use Nightscout or xDrip sync instead?

**Comment by JGroucutt** [comment]
I could use night Scout, or xdrip sync, but I cannot dictate what the
clinic I go to would like me to do. My endocrinologist wants to view data
on clarity, so I want to send treatment data to dexcom share server.

On my clinics suggestion, I've set up the BYODApp, but that does not come
with the functionality that I've come to enjoy since using xdrip+. So, I've
now got xdrip+ listening to that app as source data, but have had some
connectivity problems between my sensor and BYODApp (not your problem, ik)
which I feel were handled more automatically when I was using xdrip as the
collector.

**Reply by Navid200** [reply]
Clarity has nothing to do with Dex share. Those are two different Dexcom capabilities.
If your clinic wants you to upload to clarity and you succeed to upload to Dex share, your clinic will still not be able to see your data.

xDrip cannot upload to Clarity.

**Comment by JGroucutt** [comment]
Y not?

**Comment by janaoesau** [comment]
Hi, I have the same issue. I cannot invite any followers and data are not arriving in the dexcom g6 follower app. Always says „failed to invite followers“ after entering name and email address of an follower. 
I don‘t want data on dexcom clarity, just in the dexcom follower app. We have to follow the data of our 3 year old son when he is in the kindergarden. We are using CamAPS and xdrip in companion mode on his android mobile phone. But as me and my husband both use an iPhone and therefore xdrip sync is not possible as well as the companion mode in CamAPS itself. So we want to use the dexcom follow app for having the real-time BG values and get the alarms. And setting up Nightscout is currently too much work. Indeed we are waiting for the iOS version of the CamAPS which originally should be released end of last year but was postponed to mid of this year. that’s why I try to setup xdrip now to have the data in the dexcom follower app as an interim solution. 

Xdrip collects the data correctly from CamAPS but upload to dexcom share seems to be not working. I have checked „upload values to dexcom share“, unchecked „dexcom US account“ (as I am German), entered dexcom user name and password. I have tried several times but never working. Pretty sure username and password are correct. 
What else do I need to setup? Maybe in some other configuration options of xdrip setup?
Can someone help?

**Reply by Navid200** [reply]
We have limited number of developers.

We already have two follower schemes that are working perfectly.  Those being Nightscout and xDrip Sync.

If you tell me why neither of those two works for you and only Dex share does, may be we can understand why we need to spend time addressing issues with it.  Otherwise, the matter is not going to get any attention.

**Reply by janaoesau** [reply]
We have iPhones and for iOS xdrip and therefore xdrip synch is not available. Only the master mobile phone of our son is android

**Reply by Navid200** [reply]
Do you want to try Nightscout?
https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

**Reply by janaoesau** [reply]
I will try to setup

**Reply by Morrisen** [reply]
Got the same problem!
Daughter got android and CamAPS and we parents got Iphones and need the alarms from Dexcom share.
Installed xdrip on her phone and logged as companion app to share data through the cloud. She got data on her xdrip app but every invite to follower fails and nothing stated why.

Any tips?

**Reply by Navid200** [reply]
> Any tips?

Use Nightscout.

**Reply by topi7609** [reply]
Install original Dexcom app, invite followers, then unistall it. It works for me.
